src/platform.ts

```diff
@@ -69,14 +71,17 @@ export class LinuxDistribution {
                 else if (key === 'VERSION_ID') {
                     version = value;
                 }
+                else if (key === 'ID_LIKE') {
+                    idLike = value.split(" ");
+                }
 
                 if (name !== unknown && version !== unknown) {
```

Comment:
Everything looks good to me, but I could be missing something. What line are you thinking of?